src/riak_kv_pb_bucket.erl

```diff
@@ -94,42 +94,54 @@ encode(Message) ->
     {ok, riak_pb_codec:encode(Message)}.
 
 %% @doc process/2 callback. Handles an incoming request message.
-process(#rpblistbucketsreq{type = Type, timeout=T, stream=S}=Req, State) ->
-    lager:info("Received list_buckets job via protocol buffers"),
-    case riak_core_util:job_class_enabled(list_buckets) of
+process(#rpblistbucketsreq{type = Type, timeout = T, stream = S} = Req, State) ->
+    Class = 'list_buckets',
+    Accept = riak_core_util:job_class_enabled(Class),
+    _ = riak_core_util:report_job_request_disposition(
+            Accept, Class, ?MODULE, process, ?LINE, protobuf),
+    case Accept of
         true ->
             case check_bucket_type(Type) of
                 {ok, GoodType} ->
                     do_list_buckets(GoodType, T, S, Req, State);
                 error ->
-                    {error, {format, "No bucket-type named '~s'", [Type]}, State}
```

Comment:
Small request for the future: if/when you break up long lines, could you do it in its own separate commit? It adds a little bit of friction to the review process when I have to look at the diff and mentally filter out what's an actual change vs what's just being reformatted.